packages/core/test/interpreter.test.ts

```diff
@@ -1910,7 +1910,7 @@ Event: {\\"type\\":\\"SOME_EVENT\\"}"
       service.start();
     });
 
-    it('state.children should reference invoked child actors (observable)', (done) => {
+    it.skip('state.children should reference invoked child actors (observable)', (done) => {
```

Comment:
This test was causing an open handle which was what caused the first ci run to timeout after 20 minutes. 

I'm not familiar with observables so I'm not entirely sure what the problem here is, and whether it's just a test problem or a problem in the codebase itself. Something somewhere isn't getting cleaned up correctly though.

`--detectOpenHandles` points to this line as a potential source of the problem:

https://github.com/statelyai/xstate/blob/c4bfe4447c2190adf66eaba6f1040edb436a2fcc/packages/core/src/interpreter.ts#L1424

If `source.subscribe` creates a new subscription on each call, do we need to unsubscribe from the previous subscription before adding the new subscription?